qiskit/transpiler/layout.py

```diff
@@ -17,9 +17,8 @@
 Virtual (qu)bits are tuples, e.g. `(QuantumRegister(3, 'qr'), 2)` or simply `qr[2]`.
 Physical (qu)bits are integers.
 """
-
+from __future__ import annotations
```

Comment:
Not sure why postponed evaluation is necessary. Is it because for performance?

Reply:
Not sure what do you mean?

Reply:
why including `from __future__ import annotations`?

Reply:
It is required for `typea | typeb` syntax as well as `list[x]` instead of `List[y]` for older versions of python. e.g., line 403